cves/kernel/CVE-2021-37576.yml

```diff
@@ -373,8 +373,9 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: "Nothing about the code was changed other than the inclusion of the oob
+  check for the pointer"
 lessons:
```

Comment:
Fill out "applies: false" for those that aren't true, but we've covered in class.